### ISSUE TITLE
A11y_VisualStudioCodeClient_Customizations and settings_Changing of the Theme_Screenreader:Once the theme was updated, NVDA/Narrator did not announce any status messages.

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/vscode#159304]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG4.1.3; #NVDA; #ScreenReader; #Narrator;

### Environment and OS details:
Visual Studio Code Version: 1.70.2 (user setup)
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:

1. Turn on NVDA
2. Open Visual studio code editor.
3. Open the command Pallet by pressing "F1" key and type "Theme" and enter
4.  Verify that screen reader announcing status message once theme got changed or not.

### Actual Result:
Once the theme was updated, NVDA/Narrator did not announce any status messages.

### Expected Result:
NVDA/Narrator should announce status message for example, theme updated 

### Attachment:
![themes-status-message-nvda](https://user-images.githubusercontent.com/100189407/186915473-2ceb109f-130b-4246-9654-a56fea0a57eb.gif)
![themes-narrator](https://user-images.githubusercontent.com/100189407/186916201-578962fb-1523-49a7-bd89-91a3c08debed.gif)

**Comment by aeschli** [comment]
@isidorn I wonder if the quick pick dialog should narrate that an item has been accepted.
Otherwise, all quick pick users have to do this.

What do you think?
If you think the theme code should do this, how would this be done (API wise)?

**Comment by isidorn** [comment]
This should not be done. We currently do not confirm that an action has been completed. This is a design decision.
Closing as invalid. 0 users requested this.

@aeschli though if we were to do this it should be for all commands. Which does not make sense
